mergify_engine/tests/functional/actions/test_review.py

```diff
@@ -47,9 +47,10 @@ async def test_review(self):
 
         p, _ = await self.create_pr()
         await self.run_engine()
-
         await self.wait_for("pull_request_review", {}),
+
         await self.run_engine()
+        await self.wait_for("pull_request_review", {}),
```

Comment:
sound unrelated, maybe in a different PR? 😁

Reply:
record was failing sometime due to this.

Reply:
I understood that, which is exactly my point. This should be fixed in a different PR. 😉